catch/unit/graph/hipGraphMemsetNodeGetParams.cc

```diff
@@ -17,125 +19,65 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-    return false;
-  if (mNode1->width != mNode2->width)
-    return false;
-  return true;
-}
-
-TEST_CASE("Unit_hipGraphMemsetNodeGetParams_Functional") {
-  constexpr size_t N = 1024;
-  constexpr size_t Nbytes = N * sizeof(char);
-  constexpr size_t val = 0;
+  hipGraph_t graph = nullptr;
+  hipGraphNode_t node = nullptr;
 
-  char *devData;
-  HIP_CHECK(hipMalloc(&devData, Nbytes));
-
-  hipGraph_t graph;
-  hipGraphNode_t memsetNode;
-  HIP_CHECK(hipGraphCreate(&graph, 0));
-  hipStream_t streamForGraph;
-  HIP_CHECK(hipStreamCreate(&streamForGraph));
+  SECTION("node == nullptr") {
+    HIP_CHECK_ERROR(hipGraphMemsetNodeGetParams(nullptr, &params), hipErrorInvalidValue);
+  }
 
-  hipMemsetParams memsetParams{};
-  memset(&memsetParams, 0, sizeof(memsetParams));
-  memsetParams.dst = reinterpret_cast<void*>(devData);
-  memsetParams.value = val;
-  memsetParams.pitch = 0;
-  memsetParams.elementSize = sizeof(char);
-  memsetParams.width = Nbytes;
-  memsetParams.height = 1;
-  HIP_CHECK(hipGraphAddMemsetNode(&memsetNode, graph, nullptr, 0,
-                                  &memsetParams));
-  SECTION("Get Memset Param and verify.") {
-    hipMemsetParams memsetGetParams;
-    REQUIRE(hipSuccess == hipGraphMemsetNodeGetParams(memsetNode,
-                                                      &memsetGetParams));
-    // Validating the result
-    REQUIRE(true == memsetNodeCompare(&memsetParams, &memsetGetParams));
+  SECTION("pNodeParams == nullptr") {
+    HIP_CHECK(hipGraphCreate(&graph, 0));
+    HIP_CHECK(hipGraphAddMemsetNode(&node, graph, nullptr, 0, &params));
+    HIP_CHECK_ERROR(hipGraphMemsetNodeGetParams(node, nullptr), hipErrorInvalidValue);
+    HIP_CHECK(hipGraphDestroy(graph));
   }
-  SECTION("Set memset node params then Get and verify.") {
-    constexpr size_t updateVal = 2;
-    char *devData1;
-    HIP_CHECK(hipMalloc(&devData1, Nbytes));
-    memset(&memsetParams, 0, sizeof(memsetParams));
-    memsetParams.dst = reinterpret_cast<void*>(devData1);
-    memsetParams.value = updateVal;
-    memsetParams.pitch = 0;
-    memsetParams.elementSize = sizeof(char);
-    memsetParams.width = Nbytes;
-    memsetParams.height = 1;
 
-    hipMemsetParams memsetGetParams;
-    REQUIRE(hipSuccess == hipGraphMemsetNodeSetParams(memsetNode,
-                                                      &memsetParams));
-    REQUIRE(hipSuccess == hipGraphMemsetNodeGetParams(memsetNode,
-                                                      &memsetGetParams));
-    // Validating the result
-    REQUIRE(true == memsetNodeCompare(&memsetParams, &memsetGetParams));
-    HIP_CHECK(hipFree(devData1));
+// Disabled on AMD due to defect - EXSWHTEC-208
+#if HT_NVIDIA
+  SECTION("Node is destroyed") {
+    HIP_CHECK(hipGraphCreate(&graph, 0));
+    HIP_CHECK(hipGraphAddMemsetNode(&node, graph, nullptr, 0, &params));
+    HIP_CHECK(hipGraphDestroy(graph));
+    HIP_CHECK_ERROR(hipGraphMemsetNodeGetParams(node, &params), hipErrorInvalidValue);
```

Comment:
@music-dino : This fails on NVidia on CI. The return is hipSuccess and not hipErrorInvalidValue. So I will be disabling this section for now for all platforms.